apps/hubble/src/storage/db/types.ts

```diff
@@ -78,6 +78,9 @@ export enum RootPrefix {
 
   /* Used to index fname username proofs by fid */
   FNameUserNameProofByFid = 27,
+
+  /* Used to index reactions by target */
+  TagsByTarget = 28,
```

Comment:
Note that this needs to be exactly duplicated in the message.rs file in the Rust addon